data/sample.json

```diff
@@ -1,6 +1,7 @@
 [
   {
     "id": 0,
+    "externalId": "AAA",
```

Comment:
just curious, is this the actual external id or the id we come up?

Reply:
Just some random ID I pick up.